concordium-sdk/src/main/java/com/concordium/sdk/Client.java

```diff
@@ -325,6 +329,22 @@ public Semver getVersion() {
         return new Semver(versionString);
     }
 
+    /**
+     * Instruct the node to try to connect to the given peer.
+     * This also adds the address to the list of trusted addresses.
+     * These are addresses to which the node will try to keep connected to at all times.
+     *
+     * @param ip   Ip Address of the Node.
+     * @param port Port of the Node.
+     * @return true if Peer Connect was successful.
+     */
+    public boolean peerConnect(InetAddress ip, UInt16 port) {
```

Comment:
I suggest simply using `InetSocketAddress` and the code becomes something like: 
```
public boolean peerConnect(InetSocketAddress address) {
        return server().peerConnect(ConcordiumP2PRpc.PeerConnectRequest.newBuilder()
                .setIp(StringValue.of(address.getAddress().getHostAddress()))
                .setPort(Int32Value.newBuilder().setValue(address.getPort()).build())
                .build()).getValue();
    }
```

Reply:
done